plugin-scalebar/src/main/java/com/mapbox/maps/plugin/scalebar/ScaleBarImpl.kt

```diff
@@ -270,13 +270,20 @@ class ScaleBarImpl : ScaleBar, View {
             textPaint.textAlign = Paint.Align.LEFT
             strokePaint.textAlign = Paint.Align.LEFT
           }
+
           else -> {
             textPaint.textAlign = Paint.Align.CENTER
             strokePaint.textAlign = Paint.Align.CENTER
           }
         }
 
-        drawText(canvas, distanceText, (unitBarWidth * rectIndex), textSize)
+        drawText(
+          canvas = canvas,
+          text = distanceText,
+          x = (unitBarWidth * rectIndex),
+          y = textSize,
+          lastUnitBarText = rectIndex == rectCount
```

Comment:
```suggestion
          lastUnitBarText = (rectIndex == rectCount)
```
to better align with L283